verilog/analysis/checkers/proper_parameter_declaration_rule.cc

```diff
@@ -107,9 +112,37 @@ static const Matcher &ParamDeclMatcher() {
   return matcher;
 }
 
+void ProperParameterDeclarationRule::AddParameterViolation(
+    const verible::Symbol &symbol, const verible::SyntaxTreeContext &context) {
+  auto *token = GetParameterToken(symbol);
+
+  if (token == nullptr) {
+    violations_.insert(LintViolation(*token, kParameterMessage, context));
```

Comment:
If `token == nullptr` wouldn't dereferencing *token here kaboom ? Sounds like we just want to return early in that case

```c++
  if (!token) return;
  AutoFix ....
```

(also in the code below).

Reply:
Whoops. Fixed in both places. Thanks.